jwala-services/src/main/java/com/cerner/jwala/service/webserver/component/WebServerStateSetterWorker.java

```diff
@@ -102,15 +102,17 @@ public void pingWebServer(final WebServer webServer) {
             response = request.execute();
             LOGGER.debug("Web server {} status code = {}", webServer.getName(), response.getStatusCode());
 
-            if (response.getStatusCode() == HttpStatus.OK) {
+            if (HttpStatus.OK.equals(response.getStatusCode())) {
```

Comment:
response.getStatusCode returns the enum HttpStatus that is why equals is fine. I also tested the code before committing. Good comment though, we can never be too careful with web server status.